src/sql/transfers/evm.sql

```diff
@@ -54,3 +54,4 @@ FROM t
 LEFT JOIN contracts AS c ON c.address = t.contract
     AND ({contract:String} = '' OR c.address = {contract:String})
 WHERE isNotNull(decimals)
+ORDER BY timestamp DESC
```

Comment:
Isn't the ordering already happening here?

https://github.com/pinax-network/token-api/pull/68/files#diff-cabf8fd481ab850f9d0ec1e706ff4dffe8f39351e5d49a7cad54ae63a2fdbf67R37